File: testers/avro/scala-cats/generated-and-checked-in/com/example/events/CustomerId.scala
package com.example.events



/** Customer identifier */
case class CustomerId(value: Long) extends scala.AnyVal {
  /** Get the underlying value */
  def unwrap: Long = {
    return this.value
  }
}

object CustomerId {
  /** Create a CustomerId from a raw value */
  def valueOf(v: Long): CustomerId = {
    return new CustomerId(v)
  }
}

File: testers/grpc/kotlin-quarkus/generated-and-checked-in/com/example/grpc/CustomerId.kt
package com.example.grpc



/** Wrapper type for kotlin.String */
data class CustomerId(val value: kotlin.String) {
  /** Get the underlying value */
  fun unwrap(): kotlin.String {
    return this.value
  }

  override fun toString(): kotlin.String {
    return value
  }

  companion object {
    /** Create a CustomerId from a raw value */
    fun valueOf(v: kotlin.String): CustomerId {
      return CustomerId(v)
    }
  }
}